### PR TITLE
Add a blog about Higress

### PR DESCRIPTION
**Pull request by sjtuzbk**
Add a blog about Higress.

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-blog/SCA-Higress-Application-Deployment.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
+---
+title: Spring Cloud Alibaba应用发布实践
```
[the remark this answers is not in the document]
Comment (sjtuzbk) [reply i18n/zh-cn/docusaurus-plugin-content-blog/SCA-Higress-Application-Deployment.md]: done, thanks for reviewing!

File: i18n/zh-cn/docusaurus-plugin-content-blog/SCA-Higress-Application-Deployment.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+```
+### 请求验证
+可以看到来自安卓系统的请求被转发到了v2版本，其余系统仍访问v1版本。
+![image.png](https://intranetproxy.alipay.com/skylark/lark/0/2023/png/51256826/1687834110563-5f7d397c-a7ba-4fc4-82a9-fa6a5729a2c2.png#clientId=u9b61baf2-4bca-4&from=paste&height=280&id=ue7b2943d&originHeight=280&originWidth=3060&originalType=binary&ratio=1&rotation=0&showTitle=false&size=411876&status=done&style=none&taskId=uef2c54db-359c-4096-bf54-48f2c3eb489&title=&width=3060)
+当新版本验证完毕需要全量上线时，只需要将demo路由的`higress.io/destination`注解修改为v2版本，并删除demo-ab路由，这样所有流量就都会访问v2版本了。
+# 加入Higress和Spring Cloud Aliaba社区
+Spring Cloud Alibaba社区交流群：钉钉群号2415000986
+Higress社区交流群：
+![higress-comm.png](https://intranetproxy.alipay.com/skylark/lark/0/2023/png/22499/1687846710178-4f586c7e-4167-4031-9545-31d73679b32b.png#clientId=u6f2573e4-9612-4&from=ui&id=u2cad2756&originHeight=405&originWidth=720&originalType=binary&ratio=1&rotation=0&showTitle=false&size=95558&status=done&style=none&taskId=u6c6a9223-d0b9-4a0e-8eec-03188388463&title=)
+Higress 社区交流钉钉群中有历次 Higress 社区周会录屏，包括本文中提到的结合 Spring Cloud 应用发布的完整实操视频
```
[the remark this answers is not in the document]
Comment (sjtuzbk) [reply i18n/zh-cn/docusaurus-plugin-content-blog/SCA-Higress-Application-Deployment.md]: done

File: i18n/zh-cn/docusaurus-plugin-content-blog/SCA-Higress-Application-Deployment.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+      - backend:
+          resource:
+            apiGroup: networking.higress.io
+            kind: McpBridge
+            name: default
+        path: /version
+        pathType: Prefix
+```
+### 请求验证
+可以看到来自安卓系统的请求被转发到了v2版本，其余系统仍访问v1版本。
```
[the remark this answers is not in the document]
Comment (sjtuzbk) [reply i18n/zh-cn/docusaurus-plugin-content-blog/SCA-Higress-Application-Deployment.md]: done

**Review by chickenlj — APPROVED** [review]
LGTM.

**Pull request merged**